napari/notifications.py

```diff
@@ -0,0 +1,178 @@
+import os
```

Comment:
I know this is still WIP, but we'll probably want to put this inside `utils` and then need to add tests finish doc strings etc